source/features/show-user-top-repositories.tsx

```diff
@@ -5,20 +5,18 @@ import * as pageDetect from 'github-url-detection';
 import features from '.';
 import {getCleanPathname} from '../github-helpers';
 
-function buildUrl(queryField: string): URL {
-	const url = new URL('/search', location.href);
-	url.searchParams.set('o', 'desc');
-	url.searchParams.set('q', `user:${getCleanPathname()}`);
-	url.searchParams.set('s', queryField);
-	url.searchParams.set('type', 'Repositories');
+function buildUrl(): URL {
+	const url = new URL('/' + getCleanPathname(), location.href);
```

Comment:
This should be enough
```suggestion
	const url = new URL(location.pathname, location.href);
```